platform/viewer/src/components/Header/Header.js

```diff
@@ -1,18 +1,19 @@
-import './Header.css';
-import './Header.css';
-
-import { Link, withRouter } from 'react-router-dom';
 import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { withTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
 
 import { Dropdown } from '@ohif/ui';
-import OHIFLogo from '../OHIFLogo/OHIFLogo.js';
-import PropTypes from 'prop-types';
 import { AboutModal } from '@ohif/ui';
+
+import OHIFLogo from '../OHIFLogo/OHIFLogo.js';
 import { hotkeysManager } from './../../App.js';
-import { withTranslation } from 'react-i18next';
+import './Header.css';
+import './Header.css';
 
 // Context
 import AppContext from './../../context/AppContext';
+import { withModal } from '../../../../ui/src/utils/ModalProvider.js';
```

Comment:
This import crosses the package boundary. It would break for any non-PWA builds. You'll need to import using `@ohif/ui`

Reply:
Done! thx.!